tests/test_phase5_rebuild_ete.py

```
@@ -0,0 +1,648 @@
+    # PNG signature
+    with bloom.open("rb") as fh:
+        head = fh.read(8)
+    assert head == b"\x89PNG\r\n\x1a\n", f"bloom.png has invalid PNG header: {head.hex()}"
+
+
+# ---------------------------------------------------------------------------
+# ETE-07: autoPlaceOnce persists ac.storage flag and runs exactly once
+# ---------------------------------------------------------------------------
+
+
+def test_ete07_auto_place_once_runs_once_then_skips(lua):
```

Ruff (0.15.9)

[warning] Unused function argument: `lua`

(ARG001)